src/pages/download/_downloadPythonPackage.js

```diff
@@ -36,16 +44,40 @@ function getJSZipDateWithOffset() {
   return dateWithOffset;
 }
 
-export async function generateZIP(deviceId) {
+function handlePartialSuccess(failedImages) {
+  const description = `
+    <div>Image(s) failed to generate. Try a different image/device:</div>
+    <ul>
+      ${failedImages.map((failedImage) => `<li>${failedImage}</li>`).join("")}
+    </ul>
+    <div>If the issue persists, please report it on GitHub 🙏 <br>
+    <a href='https://github.com/oursky/mockuphone.com/issues'>https://github.com/oursky/mockuphone.com/issues</a></div>
+  `;
```

Comment:
please report it on <a href='https://github.com/oursky/mockuphone.com/issues'>Github</a>